resources/migrations/001_update_migrations.yaml

```diff
@@ -8505,6 +8505,67 @@ databaseChangeLog:
                   constraints:
                     nullable: true
 
+  - changeSet:
+      id: v51.2024-07-25T11:56:27
+      author: crisptrutski
+      comment: Wipe stale query fields, so we can add new non-nullable columns
+      rollback: # none, entries will be recreated by sweeper
+      changes:
+        - delete:
+            tableName: query_field
+            remarks: remove stale analysis, so the new fields can be non-nullable
+
+  - changeSet:
+      id: v51.2024-07-25T11:57:27
+      author: crisptrutski
+      comment: Add `column` column to query fields
```

Comment:
I'd prefer to see why this column is needed in the comment. I guess that's for "unknown" fields, right?